standalone/src/widget/index.tsx

```diff
@@ -59,6 +66,50 @@ async function initWidget({store, startingCall}: InitCbProps) {
         }
     }
 
+    window.callsClient?.on('mute', () => {
```

Comment:
This is an inconsistency I found when testing video on desktop. On the web side we do this to keep the redux state in sync with the local client. This avoids the issue of a perceivable lag between pressing mute/unmute and getting the UX to react, since otherwise we'd be waiting for a server side websocket event.